### ISSUE TITLE
Googling "python redis" returns unofficial docs

### ISSUE THREAD

**Issue by jeffwidman** [andymccurdy/redis-py#729]
Google "redis python"

Second result is here: https://redis-py.readthedocs.org/en/latest/

Seems totally legit. Then I click "edit on Github" and land on an out-of-date fork: https://github.com/WoLpH/redis-py/blob/master/docs/index.rst

It looks like this source repo has docs, so it'd be nice if they were the ones mirrored onto ReadTheDocs redis-py... especially since it's the second google result for a very generic query.

@WoLpH would you be okay transferring the `redis-py.readthedocs.org` name space to @andymccurdy ?

**Comment by WoLpH** [comment]
Yeah, no problem. At one point I added it before this repo had any docs. Transferring should be no problem at all :)

**Comment by edmorley** [comment]
This just confused me too. I don't suppose those docs could either be transferred/updated or removed? :-)

**Comment by WoLpH** [comment]
I believe @andymccurdy needs to add the readthedocs hook for this to fully work. Beyond that I believe everything should work already :)

**Comment by Grokzen** [comment]
@andymccurdy This issue could be closed. It looks like you now own the above mentioned RTD site now.

**Comment by WoLpH** [comment]
It looks like everything is set up correctly as well. At least, I see recent builds: https://readthedocs.org/projects/redis-py/builds/
